hack/onboarding.sh

```diff
@@ -98,12 +98,12 @@ fi
 echo "Creating namespace '$NAMESPACE' in the base..."
 create_namespace
 
-if [ ! -d $APP_NAME/components/rbac/$OWNER ]; then
+if [ ! -d $APP_NAME/components/project-admin-rolebindings/$OWNER ]; then
 	echo "RBAC for '$OWNER' group does not exist yet. Creating..."
 	create_project_admin_rolebinding
 fi
 
-if ! grep -q "name: $OWNER" "$APP_NAME/base/groups/groups.yaml"; then
+if ! grep -r -q "name: $OWNER" "$APP_NAME/base/groups/"; then
```

Comment:
why does it need to be recursive? is this for an edge case I'm not seeing?

Reply:
Each group is within a folder within the `base/groups` We can also use `find $APP_NAME/base/groups/ -name $OWNER -type d` or something... this was just a laziness on my end, it was a lesser change set. What would you prefer?

Reply:
Why you didn't use the same logic that you use to create the group to target that particular file. I.e.
```
mkdir -p $APP_NAME/base/groups/$OWNER
cat <<-EOF >> $APP_NAME/base/groups/$OWNER/group.yaml
```
So we know the file is always $APP_NAME/base/groups/$OWNER/group.yaml
Can't we just point it there?